code/modules/organs/organ_external.dm

```diff
@@ -598,7 +598,8 @@ This function completely restores a damaged organ to perfect condition.
 	if((damage > 15) && (type != BURN) && (local_damage > 30) && prob(damage) && (robotic < ORGAN_ROBOT) && !(species.flags & NO_BLOOD))
 		var/datum/wound/internal_bleeding/I = new (min(damage - 15, 15))
 		wounds += I
-		owner.custom_pain("You feel something rip in your [name]!", 50)
+		owner.custom_pain("You feel something rip inside of your [name], bright blood gushing from the wound!", 50) //Let's make this CLEAR that an artery was severed. This was vague enough that most players didn't realize they had IB.
+		to_chat(owner, span_bolddanger("You get the feeling that your injuries will require more than just bandaging."))
```

Comment:
This one also doesn't sit right with me. It's in like, the weird middle ground of giving info that feels like it should be explicit, but is still kind of ambiguous. I'd lean into it more with something like:

"Something in your [name]'s was torn that shouldn't have been. You're hemorrhaging blood!" - or "You're losing blood fast!" if "...you're hemorrhaging blood" is too big/complex of a word. More clear and to the point, while also hopefully leaning into the panic. The current one feels kind of sterile relative to the changes to the string above.

Reply:
Maybe ""You feel a sharp tear inside your [name], and bright blood starts gushing from the wound!"" for the one above?

Reply:
I'll have to fiddle around with this one and see what feels right.

The big thing with IB in particular is that it -often- gets drowned out when in combat. I'll be honest, I was very tempted to go /tg/ style for the description and make it do "OH GOD! SOMETHING TORE INSIDE YOUR ARM" with a pain message of 200 intensity to make it big and bold in their chat so it's _not_ missed

Reply:
Honestly that'd also work. I'm lucky to have never experienced it, but I'd imagine anything even adjacent to that would be dread-inducing enough to warrant "OH FUCK" tier text.